lang/gcc10-bootstrap/Portfile

```diff
@@ -176,21 +180,11 @@ if {${os.major} >= 18 && ${configure.sdkroot} ne ""} {
     configure.args-append --with-sysroot="[regsub {MacOSX1[0-9]\.[0-9]+\.sdk} ${configure.sdkroot} {MacOSX.sdk}]"
 }
 
-# clang (as) from Xcode 12.5 has various problems with gcc build
-if { ${os.platform} eq "darwin" && \
-         ( [ vercmp ${xcodeversion} 12.5 ] >= 0 || [ vercmp ${cltversion} 12.5 ] >= 0 ) } {
-    pre-configure {
-        ui_warn "Applying '--without-build-config' workaround to Xcode ${xcodeversion} / CLT ${cltversion}"
-        ui_warn "If versions > 12.5 please check if it is still required"
-    }
-    # gcc has build issues on macOS 11.3 with the use of Xcode clang as 'as'
-    # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100340
-    # https://trac.macports.org/ticket/62775
-    configure.args-append  --without-build-config
+# Be default Catalina and Mojave support a single arch at MacPorts
+if {${os.major} < 18 || ${os.major} > 19} {
```

Comment:
@Gcenx to continue discussion about this line. What do you think about:
```
if {${universal_possible}} {
  default_variants +universal
}
```

Reply:
If that’s your intention I believe this should be

```
if {${universal_possible} && ![variant_isset universal]} {
```

Reply:
This may have other unintended side-effects on ppc/apple silicon

Reply:
@Gcenx to be honest => this is a polish which can be done globally. Let me close this discussion here.